react/features/base/conference/reducer.js

```diff
@@ -92,6 +97,9 @@ ReducerRegistry.register(
         case SET_PASSWORD:
             return _setPassword(state, action);
 
+        case SET_PENDING_SUBJECT_CHANGE:
+            return set(state, 'pendingSubjectChange', action.subject);
```

Comment:
Does pendingSubject get left defined in redux or does it get cleared somehow? If it's left behind, it's okay it's left behind?

Reply:
It is cleared.